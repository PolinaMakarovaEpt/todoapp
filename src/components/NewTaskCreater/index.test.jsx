import React from "react";
import AddTask from "./index";
import { configure, shallow, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import { JestEnvironment } from "@jest/environment";

configure({ adapter: new Adapter() });

it("matches the snapshot", () => {
    const props = {
        onCreate: jest.fn(),
        onClick: jest.fn()
    }
    const tree = shallow(<AddTask {
        ...props
    } />);
    expect(toJson(tree)).toMatchSnapshot();
  });
