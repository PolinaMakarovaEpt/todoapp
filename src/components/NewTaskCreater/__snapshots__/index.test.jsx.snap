// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="add-task"
>
  <input
    autoFocus={true}
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Введите дело"
    value=""
  />
  <button
    id="add"
    onClick={[Function]}
  >
    Добавить
  </button>
</div>
`;
